dutch/over-het-hek/hek.md

```diff
@@ -0,0 +1,5 @@
+... vol overgave neem je een aanloop en springt over het hek. Met een elegante zwaai vlieg je eroverheen.
+
+Nou ja... bijna dan. Net voordat je voeten de grond raken kom je plots tot stilstand. De tranen springen je in de ogen, want het gevoel van hoe je kleren plotseling in je vel knijpen is niet bepaald prettig.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.